search/src/Components/upload/upload_stepper.js

```diff
@@ -246,11 +248,20 @@ class UploadStepper extends React.Component {
         })
     }
 
+    handleCocoFormValidation(formKey, status){
+        this.setState(prevState => {
+            const newState = {coco_form_validation: {...prevState.coco_form_validation}}
+            newState.coco_form_validation[formKey] = status
+            console.log(newState)
+            return newState
+        })
+    }
+
     handleExempt(){
```

Comment:
I still find this name hard to understand. Rename it?

Reply:
checkActiveStepExemptible?

Reply:
> `checkActiveStepExemptible`

I don't know what it's exempt from, from the name. Should it just be named something like `isNextEnabled`?

Reply:
Cause the next buttons were all disabled by default, except for forms. Never mind for now. I'll name it as isNextEnabled